libp2p/NodeTable.cpp

```diff
@@ -423,7 +428,7 @@ void NodeTable::onReceived(UDPSocketFace*, bi::udp::endpoint const& _from, bytes
                     auto e = m_evictions.find(in.sourceid);
                     if (e != m_evictions.end())
                     { 
-                        if (e->second.evictedTimePoint > std::chrono::steady_clock::now())
+                        if (e->second.evictedTimePoint + c_reqTimeout > std::chrono::steady_clock::now())
```

Comment:
To be symmetrical to the check in `doCheckEvictions` it should be `>=` 
```suggestion
                        if (e->second.evictedTimePoint + c_reqTimeout >= std::chrono::steady_clock::now())
```